spec/system/feedback_spec.rb

```diff
@@ -5,6 +5,7 @@
     sign_in create(:caseworker)
     visit root_path
     click_on 'Feedback'
+    ActiveJob::Base.queue_adapter.enqueued_jobs.clear
```

Comment:
Nit: this feels like something that, if it's needed, should be handled by a `config.after(:each, type: :system) { ... }` block in the rails_helper file so that we don't need to remember to do it in every system spec where it matters.